dspace-api/src/test/java/org/dspace/eperson/GroupTest.java

```diff
@@ -206,6 +206,9 @@ public void findAllNameSort() throws SQLException {
         List<String> names = new ArrayList<>();
         List<String> sortedNames = new ArrayList<>();
         for (Group group : groups) {
+            if (group.getName() == null) {
+                continue;
```

Comment:
Why would we want to support un-named groups?  Shouldn't it be made impossible to persist a group with no name?